lib/matplotlib/_api/deprecation.py

```diff
@@ -16,6 +16,42 @@
 import warnings
 
 
+# has to be here (instead of cbook) to prevent circular import, since
```

Comment:
It will never be in `cbook` again; should this say `_api`, or is the comment now irrelevant?

Reply:
Yeah sorry, this comment was from before the `_api` changes, and should definitely read `_api`, not cbook.